docs/Installation.md

```diff
@@ -16,7 +17,7 @@ Python 3.6 or higher is required.
 $ pip install quilt3[pyarrow]
 ```
 
-If you do not need to serialize and deserialize dataframes with Quilt, you can
+If you do not need to serialize and deserialize dataframes with Quilt <!-- SK: How do I know? --> , you can
```

Comment:
we should probably extract all the docs changes out of this PR -- they are out of scope